tests/ert_tests/gui/models/test_job_list_model.py

```diff
@@ -27,9 +34,7 @@ def test_using_qt_model_tester(qtmodeltester, full_snapshot):
     model.setSourceModel(source_model)
 
     reporting_mode = qt_api.QtTest.QAbstractItemModelTester.FailureReportingMode.Warning
-    tester = qt_api.QtTest.QAbstractItemModelTester(  # noqa, prevent GC
-        model, reporting_mode
-    )
```

Comment:
Look at the comment here, please revert this

Reply:
I think perhaps the comment is not sufficient. Should've been something like "the tester needs to be kept in the stack for the duration of the test" or something.

Reply:
Went a bit fast, my linter in my IDE complained though (not sure why) so I figured I would fix it..